BUILD.boost

```diff
@@ -323,7 +323,13 @@ boost_library(
 
 boost_library(
     name = "asio",
-    defines = select({
+    srcs = [
+        "boost/asio/impl/src.cpp",
+        "boost/asio/impl/src.hpp",
```

Comment:
Sorry for very late question. Trying to understand this rule, what does the srcs array mean? In particular, any reason whey the same source appears twice?

Reply:
It specifies the inputs to this rule. Note that it's not the same file; one is `.cpp` and one is `.hpp`.

Reply:
Oh my. I looked it over several times, and even followed the documentation link, somehow still missed that. Time for break. Thanks for the reply!